auth0/resource_auth0_connection_test.go

```diff
@@ -510,18 +510,21 @@ resource "auth0_client" "my_client_1" {
 }
 
 resource "auth0_client" "my_client_2" {
+	depends_on = [auth0_client.my_client_1]
```

Comment:
There are cases where we need to introduce the depends_on meta argument in order to create a sequential execution of requests for our interactions. Otherwise when multiple resources of the same type are present and there's no dependency between them, their creation will get executed in a random order as it will be done in go routines. We needed to introduce this to make our tests predictable.